### PR TITLE
feat(oui-calendar): add oui-calendar component

### PR DESCRIPTION
**Pull request by AxelPeter**
UU-562
http://build_oui_calendar.ui-kit.ovh/#!/oui-angular/calendar

Related to
https://github.com/ovh-ux/ovh-ui-kit/pull/172
https://github.com/ovh-ux/ovh-ui-kit-documentation/pull/77

### CODE REVIEW

File: packages/oui-calendar/README.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+# Calendar
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+**This component need [flatpickr](https://github.com/flatpickr/flatpickr) to work.**
```
Comment (neolitec) [inline packages/oui-calendar/README.md]: _needs_

File: packages/oui-calendar/README.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+```html:preview
+<oui-calendar model="$ctrl.disableDateModel" disable-date="$ctrl.disableDate">
+</oui-calendar>
+<div class="oui-doc-preview-only">
+    <p><strong>disableDate value:</strong> {{$ctrl.disableDate | json}}</p>
+</div>
+```
+
+**Note**: See [Flatpickr documentation](https://flatpickr.js.org/examples/#disabling-dates) for more informations.
```
Comment (neolitec) [inline packages/oui-calendar/README.md]: _information_

File: packages/oui-calendar/README.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+```html:preview
+<oui-calendar model="$ctrl.formatModel" format="d-m-Y" alt-format="F j, Y">
+</oui-calendar>
+<div class="oui-doc-preview-only">
+    <p><strong>Model value:</strong> {{$ctrl.formatModel | json}}</p>
+</div>
+```
+
+**Note**: See [Formatting Tokens](https://flatpickr.js.org/formatting/) for more informations.
```
Comment (neolitec) [inline packages/oui-calendar/README.md]: _information_

File: packages/oui-calendar/README.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    <p><strong>Model value:</strong> {{$ctrl.eventsModel | json}}</p>
+    <p><strong>onChange values:</strong> {{$ctrl.onChangeValues | json}}</p>
+    <p><strong>onOpen values:</strong> {{$ctrl.onOpenValues | json}}</p>
+    <p><strong>onClose values:</strong> {{$ctrl.onCloseValues | json}}</p>
+</div>
+```
+
+**Note**: If you want to access the parameters inside events callback, you need to use `selectedDates` or `dateStr` variables as below.
+
+* `selectedDates` return an array of Date objects selected by the user. When there are no dates selected, the array is empty.
```
Comment (neolitec) [inline packages/oui-calendar/README.md]: _returns_ ?

File: packages/oui-calendar/README.md (hunk 1 above)
Comment (AxelPeter) [reply packages/oui-calendar/README.md]: done :)

File: packages/oui-calendar/README.md (hunk 2 above)
Comment (AxelPeter) [reply packages/oui-calendar/README.md]: done for all :)

File: packages/oui-calendar/README.md (hunk 4 above)
Comment (AxelPeter) [reply packages/oui-calendar/README.md]: done for all :)

**Pull request merged**